app/services/getbookbyid_service.rb

```diff
@@ -0,0 +1,22 @@
+class GetbookbyidService < ApplicationService
+  require "net/http"
+  require "json"
+
+  Base_URL = "http://openlibrary.org/"
```

Comment:
As you can notice we have moved the duplication of base_url from one place to multiple places now. Instead of defining these here what you can do is you can define these inside `environments.development.rb` like

``` ruby
# environment/development.rb
config.OPENLIB_BASE_URL = "http://openlibrary.org"
```

When you want to use you you can use like following
``` ruby
Rails.application.config.OPENLIB_BASE_URL
```

As you can see, If you have a use case of different variables depending on your environment like `production`, `development`. You can achieve that by defining same variable in different files under `environment/<ENV>.rb`

Reply:
yes, I was thinking of doing that, only I didn't know how to do that in rails. Will do it soon.